openapi/components/schemas/RichBillingPortal.yaml

```
@@ -0,0 +1,140 @@
+type: object
+required:
+  - name
+  - recurringOptions
+  - levels
+  - solutions
+properties:
+  id:
+    type: string
+    description: ID of the billing portal.
+    readOnly: true
+    maxLength: 50
+    example: bill_prt_0YV7K5TYV5D1P9ZNKDT39KZC3C
+  name:
+    type: string
+    description: Billing portal name.
+  recurringOptions:
+    type: array
+    description: Recurring options for the billing portal. For example, it can have `monthly` and `yearly` variants.
+    items:
+      type: object
+      properties:
+        id:
+          type: string
+          description: |-
+            ID of a recurring options.
+            It must be unique between other recurring options in the same billing portal.
```

suggestion: for readability
```suggestion
            This value must be unique between other recurring options in the same billing portal.
```